### ISSUE TITLE
Web Components Swag

### ISSUE THREAD

**Issue by zenorocha** [webcomponents/webcomponents.github.io#62]
Hey guys,

What do you think about Web Components stickers and t-shirts? :)

@brizabueno & @bernarddeluna, can you propose some stuff?

In the meanwhile @addyosmani will ask around and find out who created the original logo so we can get sign-off to use it for those purposes.

**Comment by Nevraeka** [comment]
+1 on the t-shirt & sticker swag. I don't have vector versions of each spec but here is a concept for @domenic 's comment. I love the idea of separate spec shirt 

![screen shot 2015-01-14 at 9 04 37 am](https://cloud.githubusercontent.com/assets/647043/5739927/8f817828-9bcc-11e4-8f36-9b73a2ebd889.png)

**Comment by Nevraeka** [comment]
Here is asimilar one for CE - 
![screen shot 2015-01-15 at 7 08 32 am](https://cloud.githubusercontent.com/assets/647043/5757115/d171d160-9c85-11e4-893c-5cca20447a1a.png)

**Comment by zenorocha** [comment]
That's awesome @Nevraeka!

I think there's room for some design improvements but I really liked the phrases.

**Comment by Nevraeka** [comment]
@zenorocha - yeah. 

HTML Imports - Distributed with love
Templates - Parsed but not Rendered

**Comment by zenorocha** [comment]
Cool!

**Comment by zenorocha** [comment]
Hey guys!

I just published a new page called [Swags](http://webcomponents.org/swags/) to feature our stickers produced by [StickerMule](http://www.stickermule.com/marketplace/2731-web-components/) and a new set of shirts produced by [Cotton Bureau](https://cottonbureau.com/products/web-components).

The reason why I chose them to kickstart producing our shirts is because they have a pretty good quality and handle pretty much everything.

I know we've been discussing several design options and I hope we can learn from this first version so we produce them in the future :)
